mungegithub/features/repo-updates.go

```diff
@@ -236,6 +237,17 @@ func (o *RepoInfo) LeafAssignees(path string) sets.String {
 	return peopleForPath(path, o.assignees, true)
 }
 
+// ReadFile returns the contents of a file in the checked out copy of the
+// repository
+func (o *RepoInfo) ReadFile(path string) (string, error) {
+	bytes, err := ioutil.ReadFile(path)
```

Comment:
Is this entire function useful? I was expecting it to modify the path. Why wouldn't people just call ioutil.ReadFile if they want to do this?